### DISCUSSION TITLE
[DEV] Any reason why instanced Bosses don't have a boss rank in DB?

### DISCUSSION THREAD

**Discussion by BarbzYHOOL** [azerothcore/azerothcore-wotlk#3959] [General]
Only outdoor bosses have a boss rank (=3), why not mark the instanced bosses as well? (raids, dungeons, alterac valley)

All bosses with a boss rank:
```sql
SELECT entry,name,minlevel,subname, DamageModifier, rank
FROM `creature_template`
WHERE `rank` = 3
```

**Comment by ghost** [comment]
Probably because a new rank wasn't needed yet... What would you like to have a rank for bosses for?

**Comment by BarbzYHOOL** [comment]
to select them all and do some stuff with them
